### PR TITLE
Update CLI Libraries doc to .NET Standard

### PR DESCRIPTION
**Pull request by cartermp**
/cc @richlander @rpetrusha @mairaw

### CODE REVIEW

File: docs/core-concepts/libraries/libraries-with-cli.md [hunk 1]
```diff
@@ -179,7 +184,7 @@ So, the `project.json` file could look like this:
                 "System.Threading.Tasks":""
             }
         },
-        "netstandard1.5":{
+        "netstandard1.6":{
             "dependencies": {
                 "NETStandard.Library":"1.5.0-rc2",
```
Comment (blackdwarf) [inline docs/core-concepts/libraries/libraries-with-cli.md]: Change the dependency version to be RTM pls.

File: docs/core-concepts/libraries/libraries-with-cli.md [hunk 2]
```diff
@@ -298,7 +304,7 @@ For example, if you want to target [PCL profile 328](http://embed.plnkr.co/03ck2
 ```json
 {
     "frameworks":{
-        "netstandard1.5":{
+        "netstandard1.6":{
            "dependencies":{
                 "NETStandard.Library":"1.5.0-rc2-final",
```
Comment (blackdwarf) [inline docs/core-concepts/libraries/libraries-with-cli.md]: Same here, RTM version. I believe it is just "1.6.0".

**Comment by mairaw** [comment]
LGTM modulo the feedback already left by @blackdwarf

**Comment by blackdwarf** [comment]
Now, the general thing is that we need to start removing the "CLI" all over the place and replacing it with ".NET Core Tools" or similar. @richlander what is the guidance here?

**Comment by BillWagner** [comment]
Why do I think I'll get really good at 'sed' again?

**Comment by cartermp** [comment]
@BillWagner will that be a repo-wide change?  If so, then I think we can just merge this in.

**Comment by blackdwarf** [comment]
@BillWagner khm, perl, khm...

**Comment by BillWagner** [comment]
@cartermp Yes, just merge this in. 

I'm thinking that as I go through all the samples there will be a few global changes in the sample code.

**Comment by cartermp** [comment]
Let it be done!

**Pull request merged**